sisl/viz/plotly/plot.py

```diff
@@ -1664,7 +1661,7 @@ def clone(self, *args, **kwargs):
 
     def dispatch_event(self, event, *args, **kwargs):
         """ Not functional yet """
-        print(event, args, kwargs)
+        warn((event, args, kwargs))
```

Comment:
Hahaha this was just for debugging while developing, it should be removed. I was beggining to experiment with responding to clicks and other events, but didn't do it in the end. I'm looking forward to set up events in plots by the way!

Reply:
Ok, I think you can just delete it in the clean up then :)